tox.ini

```diff
@@ -0,0 +1,44 @@
+[tox]
+downloadcache = {toxworkdir}/_download/
+envlist = py27-dj18,coverage,pep8,pylint
+
+[testenv]
+commands = {envpython} manage.py test
+
+[testenv:py27-dj18]
+deps =
+    -rrequirements.txt
```

Comment:
1. In the tox.ini in three places 
Or 
2. Once in requirements and then gets installed for pep8 and pylint when it's not needed.

(1) is more efficient for loading and (2) is more efficient for running.  I'd opt to put it in the needed test in the tox.ini